src/generators/static-theme.ts

```diff
@@ -249,11 +250,23 @@ export function formatStaticThemes(staticThemes: StaticTheme[]) {
     });
 }
 
-function getCommonTheme(themes: StaticTheme[]) {
-    return themes[0];
+// TODO: do not rely on common theme being the first in the file
+function getCommonTheme(staticThemes: string, staticThemesIndex: SitePropsIndex<StaticTheme>): StaticTheme {
```

Comment:
I'm a bit worried that code relies on the assumption that the first rule is for every domain (has pattern `*`).

Reply:
Then I'll just remove the comment and add a unit test wich ensures published configs have `*` rule.

Reply:
Turns out, there is already test for this. I'll just remove the comment then.
https://github.com/darkreader/darkreader/blob/58416b36d7d0ce1dd7700e01b8e56c1bcbdb226a/tests/config/config.tests.ts#L57